src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,42 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
+const sexData = [
+  { id: 1, title: 'All', lable: '' },
+  { id: 2, title: 'Male', lable: 'm' },
+  { id: 3, title: 'Female', lable: 'f' },
```

Comment:
There is a typo in the 'sexData' array: 'lable' should be corrected to 'label'. This will ensure consistency and prevent potential issues.